Packages/Status/Sources/Status/Editor/Components/StatusEditorAccessoryView.swift

```diff
@@ -35,6 +38,15 @@ struct StatusEditorAccessoryView: View {
         } label: {
           Image(systemName: "number")
         }
+
+        Button {
+          withAnimation {
+            viewModel.showPoll.toggle()
+          }
+        } label: {
+          Image(systemName: "chart.bar")
+        }
+        .disabled(isShowingPoll)
```

Comment:
Why is this disabled when showing the poll? Couldn't it also hide the poll?
It should be disabled if there are medias attached though.

Reply:
The poll icon is disabled to help make the intention of creating a poll clear. However, we can run into an issue where the user taps the icon, fills in some information, taps it again to hide it then hits the post. 
- Was the intention to post with or without a poll? 
- Did they mean to dismiss or throw away the poll by hiding it? 
- Did they forget about the poll and mean to post it?

Reply:
Good catch for your second point. When I get back in front of my computer, I will update the code to disable this.